conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/TezosGovernanceOperations.scala

```diff
@@ -193,15 +197,15 @@ object TezosGovernanceOperations extends LazyLogging {
    * extract the numbers returning a collector object.
    */
   private def fillAggregates(
-      proposalsBlocks: Map[Block, ProtocolId],
+      proposalsBlocks: Map[Block, Option[ProtocolId]],
```

Comment:
`listingsPerLevel` is still too generic a name.

Reply:
I've renamed those as `rollsByLevel` both here and in the code that pass them through